src/test/java/plugins/MetricsTest.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2023 CloudBees, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package plugins;
+
+import org.jenkinsci.test.acceptance.AbstractPipelineTest;
+import org.jenkinsci.test.acceptance.junit.WithPlugins;
+import org.jenkinsci.test.acceptance.po.Build;
+import org.jenkinsci.test.acceptance.update_center.PluginSpec;
+import org.junit.Test;
+import org.openqa.selenium.TimeoutException;
+
+import java.io.IOException;
+import java.util.concurrent.Callable;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.regex.Pattern;
+
+import static org.junit.Assert.assertThat;
+import static org.jenkinsci.test.acceptance.Matchers.hasContent;
+
+public class MetricsTest extends AbstractPipelineTest {
+
+    private static final String METRICS_URL = "metrics/currentUser/%s";
+    private static final String NOT_EXISTENT_NODE = "not_existent_node";
+
+    private final Logger LOGGER = Logger.getLogger(MetricsTest.class.getName());
+
+
+    @Test
+    @WithPlugins("metrics")
+    public void testMetrics() throws IOException {
+        this.checkHealthcheck();
+        this.checkPing();
+        this.checkMetrics(0);
+
+        // install a new plugin and create a job which will be queued forever
+        jenkins.getPluginManager().installPlugins(new PluginSpec("parameter-separator", null));
```

Comment:
Does the plugin need to be installed at this point? If not, can we put it into `@WithPlugins`?